src/Text/XML/DSig.hs

```diff
@@ -250,7 +250,7 @@ verifyIO creds el signedID = capture' $ do
 verifyIO' :: SignCreds -> LBS -> String -> IO (Either HS.SignatureError ())
```

Comment:
Title of PR might be misleading, as it's not just testing that is affected. Here the implementation of verification changes